kempner_hpc_handbook/storage_and_data_transfer/data_transfer.md

```diff
@@ -104,6 +104,40 @@ This behaves differently. Now, the files that were in the results directory on t
 
 ````
 
+(fpsync_section)=
+## fpsync
+While `rsync` is appropriate for most synchronization needs, `fpsync` may be more suitable for synchronizing large directories. `fpsync` synchronizes directories in parallel by using `fpart` and `rsync` to launch several jobs simultaneously, and, like `rsync`, the synchronization can be resumed if disrupted. `fpsync` can launch synchronization processes locally or remotely on one or more workers using ssh. Remote workers must be able to access both the source and destination directories. Unlike `rsync`, only the source directory contents are synchronized, not the directory itself.
+
+```bash
+fpsync [arguments] [path to source directory] [path to destination directory]
```

Comment:
Please change [path to the source directory] to [absolute path to the source directory]

Reply:
Also the target path.